packages/website/docs/Slider.md

```diff
@@ -1,33 +1,39 @@
 
--   For binary settings, use a [slide toggle](#/styles/form-controls/slide-toggle), [toggle](#/components/FlatSelect), or [checkbox](#/components/Checkbox) instead.
+A slider provides a visual cue to the users on the full range of the value to be set. Use a slider when real-estate is not an issue and when visualizing the range helps users make more informed choices.
 
-### Labels
+The slider should have a range of up to 100 values. If your value range is extremely short (5 or less) or extremely large (over 50), consider using a [numeric input](#/form/NumericInput) instead.
 
--   In most cases, Sliders should have a label.
+Use sliders carefully, as they can be difficult to control on smaller devices or when the range is very large but the space is limited.
 
--   Use the shortest mark labels possible, ideally one word (e.g., Lower, Middle, Higher).
+## Labeling
 
-### Behavior
+Keep marker labels short, preferably **under only one word**.
 
--   Adjust the size and behavior of the rail so that users have access to the precision level they need, no more, no less.
+Provide a short descriptive title without action verbs. For example, write "Thumbnail size" rather than "Select the size of the thumbnail".
 
--   Always set the lowest value on the left side and the highest on the right.
+## Track and markers
 
--   You may include labeled marks to help users select discrete values (e.g., Lower, Middle, Higher). Jumping to those marks is possible by clicking on them or on their labels.
+The track is the representation of the range. It should be divided into equal steps using markers.
 
----
+Name markers using adjectives when exposing the numeric value is unimportant. For instance, use “small” and “big” when choosing the size of a thumbnail.
 
-## Related Components
+When using numeric value to label markers, display the value unit to provide context over the type of value. For instance, show "90%" if setting a percentage. 
+
+Use positive and negative symbols as prefixes when the range allows negative values.
+
+## Feedback and Validation
 
-[Toggle](#/components/FlatSelect)
+A slider always has a default value. Consider the most frequently set value as default first. If none particularly stands out, consider using a neutral value or the median. Note that the default value should stand on a marker as much as possible.
 
-[Radio Button](#/components/Radio)
+If a **numeric** value between two markers can be chosen, display the selected value above the handle or next to the slider so users can easily refer to what they have selected.
 
-[Dropdown List](#/components/DropdownSearch)
+---
+
+## Related Components
 
-[Slide Toggle](#/styles/form-controls/slide-toggle)
+If your use case doesn't match the guidelines above, consider using one of the following components instead:
 
-[Checkbox](#/components/Checkbox)
+-   [Numeric input](#/form/NumericInput) when setting a precise value is important.
```

Comment:
If you accepted my previous comment, delete this.

Reply:
@kpoulincoveo

Reply:
Accepted @aiheon ready to merge